### DISCUSSION TITLE
Auto color for rides with multiple cars

### DISCUSSION THREAD

**Discussion by jcurbaniak** [OpenRCT2/OpenRCT2#22198] [Feature requests] [answered]
For rides with multiple cars and each car different car option turned on,  provide an option to random color each car.    Having to change the color for each car manually is tedious.

**Comment by MASHzj** [comment]
That would be nice especially for bumper cars, as those have mostly a different colour per car in real life.

**Comment by mrmagic2020** [comment]
I managed to implement this feature locally. If it is approved I will make a PR for it :)

https://github.com/OpenRCT2/OpenRCT2/assets/104749673/33dc432e-81f2-4246-94ab-1b008cbc50ca

**Reply by mrmagic2020** [reply, marked as answer]
PR submitted: #22206 :)

**Reply by Gymnasiast** [reply]
> That looks good! Why wouldn't you submit a PR? The team can still decide then if they want to approve it or not.

We’ll decide on that policy. I have explicitly told @mrmagic2020 (as well as other contributors) to always ask us first in order to avoid wasting their time and ours.

**Comment by J-PIE-314** [comment]
This feature was implemented in PR #22206.

**Comment by jcurbaniak** [comment]
Thank you.   It is wonderful.   Saves a lot of time.

 

Jerry Urbaniak
